public/expressions/helpers.ts

```diff
@@ -31,22 +31,32 @@ import { get } from 'lodash';
 export const getAnomalies = async (
   detectorId: string,
   startTime: number,
-  endTime: number
+  endTime: number,
+  resultIndex: string
 ): Promise<AnomalyData[]> => {
   const anomalySummaryQuery = getAnomalySummaryQuery(
     startTime,
     endTime,
     detectorId,
     undefined,
-    false
-  );
-
-  const anomalySummaryResponse = await getClient().post(
-    `..${AD_NODE_API.DETECTOR}/results/_search`,
-    {
-      body: JSON.stringify(anomalySummaryQuery),
-    }
+    true
```

Comment:
Don't forget to revert this again back to `false`

Reply:
done